.gitattributes

```diff
@@ -0,0 +1 @@
+src/vmware/azext_vmware/tests/latest/recordings/*.yaml linguist-generated=true
```

Comment:
May I ask why this file is submitted?

Reply:
It indicated the the recordings are generated. It makes reviewing pull requests easier in GitHub by not rendering generated files by default:

![image](https://user-images.githubusercontent.com/87583576/129084591-be6c6ffc-763a-44e4-8da6-da87bed1a910.png)